examples/inverse/plot_snr_estimate.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+"""
+============================
+Plot an estimate of data SNR
+============================
+
+This estimates the SNR as a function of time for a set of data.
+"""
+# Author: Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+from os import path as op
+import numpy as np
+
+from mne.datasets.sample import data_path
+from mne.minimum_norm import read_inverse_operator
+from mne import read_evokeds
+from mne.viz import plot_snr_estimate
+
+print(__doc__)
+
+data_dir = op.join(data_path(), 'MEG', 'sample')
+fname_inv = op.join(data_dir, 'sample_audvis-meg-oct-6-meg-inv.fif')
+fname_evoked = op.join(data_dir, 'sample_audvis-ave.fif')
+
+inv = read_inverse_operator(fname_inv)
+evoked = read_evokeds(fname_evoked)[0]
+evoked.data -= evoked.data[:, evoked.times <= 0].mean(1)[:, np.newaxis]
```

Comment:
what's the purpose of this line? why not use our baseling functions? ` mne.baseline.rescale`